db.go

```diff
@@ -69,18 +69,21 @@ func optionsToParams(opts ...map[string]interface{}) (url.Values, error) {
 
 // rowsQuery performs a query that returns a rows iterator.
 func (d *db) rowsQuery(ctx context.Context, path string, opts map[string]interface{}) (driver.Rows, error) {
+	keys := opts["keys"]
+	if keys != nil {
+		delete(opts, "keys")
+	}
```

Comment:
I think this `if` can be removed, as `delete` is safe even if they key doesn't exist.